tests/sfko/sfko/report.py

```diff
@@ -0,0 +1,151 @@
+import datetime
+import json
+import sched
+from collections import namedtuple, OrderedDict
+from itertools import chain
+from threading import Thread
+
+import requests
+
+from . import execute
+from .util.conf import config
+from .util.log import Log
+from .util.redis import redis_client
+
+_log = Log('report')
+
+
+def utc_time():
+    datetime.datetime.utcnow().timestamp()
+
+class Scheduler(Thread):
+    def __init__(self):
+        super().__init__()
+        self.daemon = True
+        self._scheduler = sched.scheduler()
+
+    def run(self):
+        while True:
+            self._scheduler.run()
+
+    def enter(self, *args, **kwargs):
+        return self._scheduler.enter(*args, **kwargs)
+
+_RESULTS_SCHEDULER = Scheduler()
+
+__hour, __minute = config.results.schedule.split(':')
+_RESULTS_SCHED_TIME = datetime.time(hour=int(__hour), minute=int(__minute))
```

Comment:
upper case should be reserved for constants only (and preferably on top of file or class), here it looks like a variable so should be lowercase (though the line using it is commented so maybe bound to disappear?)